helm_deploy/values-preprod.yaml

```diff
@@ -17,12 +17,13 @@ generic-service:
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
     IN_MAINTENANCE_MODE: false
+    ADD_NOTES_DISABLED: 'true'
```

Comment:
minor again but I'd prefer a more boolean-y name, the `ADD` makes my brain stumble - `NOTES_ARE_DISABLED`, `NOTES_DISABLED`?

Reply:
good point, have changed this :)